soccer/Robot.hpp

```diff
@@ -109,6 +113,25 @@ class Robot: public RobotPose
 		return *this == other;
 	}
 
+	std::string to_string() const {
```

Comment:
We need to pick a convention and stick to it for naming things.  Most everything else is camel case, so let's make this `toString()`.